doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -30,7 +30,20 @@ To understand the following content better, you should understand the difference
   * `rayClusterSpec` - Defines the **RayCluster** custom resource to run the Ray job on.
 * Ray job configuration
   * `entrypoint` - The submitter runs `ray job submit --address ... --submission-id ... -- $entrypoint` to submit a Ray job to the RayCluster.
-  * `runtimeEnvYAML` (Optional): A runtime environment that describes the dependencies the Ray job needs to run, including files, packages, environment variables, and more. Provide the configuration as a multi-line YAML string. See {ref}`Runtime Environments <runtime-environments>` for more details. _(New in KubeRay version 1.0.0)_
+  * `runtimeEnvYAML` (Optional): A runtime environment that describes the dependencies the Ray job needs to run, including files, packages, environment variables, and more. Provide the configuration as a multi-line YAML string. Example:
+
+    ```yaml
+    # [...]
```

Comment:
Not sure if this adds any value. Is there some pseudo comment that would make this more useful? If not and this is common practice, then please disregard my comment.

Reply:
Not sure either. I don't feel strongly about it so I removed it